widget/Toaster.js

```diff
@@ -79,7 +81,8 @@ define([
 			this.ownerDocument.body.appendChild(this.domNode);
 
 			if(this.messageTopic){
-				connect.subscribe(this.messageTopic, this, "_handleMessage");
+				// TODO: wrap in a call to this.own
+				this.own(topic.subscribe(this.messageTopic, lang.hitch(this, "_handleMessage")));
```

Comment:
Yes, missed it, thanks. Part of me feels like not using this.subscribe as it feels wrong that it's there, but that's perhaps silly. :)